plugins/chat/assets/javascripts/discourse/components/chat-message.hbs

```diff
@@ -46,6 +46,7 @@
   {{on "touchstart" this.handleTouchStart passive=true}}
   {{on "touchend" this.handleTouchEnd passive=true}}
   {{on "mouseenter" (fn @onHoverMessage @message (hash desktopOnly=true))}}
+  {{on "mousemove" (fn @onHoverMessage @message (hash desktopOnly=true))}}
```

Comment:
This is a consequence of the longer delay on scroll ended. Without this change your mouse could end up on a message at the end of the scroll and we would still not consider the scroll as ended and wouldnt trigger hovering (which is the whole of this optimization, not trigger mouseover during scroll as it's expensive). Now as soon as you move the mouse we try to hover again.